crates/pathfinder/src/bin/pathfinder/main.rs

```diff
@@ -92,7 +95,12 @@ Hint: This is usually caused by exceeding the file descriptor limit of your syst
     // Set the rpc file connection limit to a fraction of the RPC connections.
     // Having this be too large is counter productive as disk IO will then slow down
     // all queries.
-    let rpc_storage = std::cmp::max(10, config.max_rpc_connections.get() / 8);
+    let max_rpc_connections: u32 = config
+        .max_rpc_connections
+        .get()
+        .try_into()
+        .expect("usize should cast to u32");
+    let rpc_storage = std::cmp::max(10, max_rpc_connections / 8);
```

Comment:
I had to choose between one of the two -- and one would always have a cast regardless. However, I figured it made sense to keep the type as defined by the RPC crate 🤷‍♂️ Essentially just keep the cast as high up as possible. That said, fairly sure this happened just because I was refactoring from the RPC crate upwards i.e. this main / config change happened towards the end.